### ISSUE TITLE
Subscription Transition to Unregistered

### ISSUE THREAD

**Issue by paeppers** [Azure/azure-resource-manager-rpc#81]
For page Subscription Lifecycle API reference,

For transitions to Unregistered state, if the RP has active resources under a subscription and receives a notification to unregister, should the RP reject the request by returning an error response? Or when the RP receives a subscription notification, should the RP return a success response while silently raising errors internally to get incident mitigation engaged?

In other words, which source of truth wins out here
